mix.exs

```diff
@@ -5,8 +5,8 @@ defmodule Amqpx.MixProject do
     [
       app: :amqpx,
       name: "amqpx",
-      version: "5.5.0",
-      elixir: "~> 1.7",
+      version: "5.6.0",
+      elixir: "~> 1.12",
```

Comment:
sono indeciso su questo. cosa ne pensate?

Reply:
io lo abbasserei, perchè se no non si può usare con versioni minori di elixir, credo vada proprio in errore

Reply:
lo lasceresti alla 1.7 come era prima?